src/components/MiniMap/MiniMap.tsx

```diff
@@ -43,7 +39,7 @@ const MiniMap: FunctionComponent<MiniMapProperties> = ({
             selected={mapOption === MAP_FILTERS.STATE}
             onClick={() => setMapOption(MAP_FILTERS.STATE)}
           >
-            {stateName}
+            {region.abbreviation}
```

Comment:
This will take it from the full state name to the abbreviation - this should probably be `fullName`

Reply:
I think it should be `shortName`, `fullName` might be too long for MSAs